### DISCUSSION TITLE
oneMPI / MPI for pre-compiled radioss download

### DISCUSSION THREAD

**Discussion by danFrodo** [OpenRadioss/OpenRadioss#1364] [Q&A] [answered]
Hi folks.  I started a new job and trying to run openradioss on windows laptop.  I downloaded the pre-compiled version and it runs perfectly.   I followed the advice here to download & install oneMpi but I can't call it because it doesn't exist.  THe install wizard runs & completes with no issues but the "C:\Program Files (x86)\Intel\oneAPI" directly simply doesn't exist.  
Anyone seen this before?  

If I am running a pre-complied download, what MPI / oneMPI stuff do actually need?

**Comment by danFrodo** [comment]
NOTE to above:  runs perfectly on one processor, trouble getting it going w multiple

**Comment by danFrodo** [comment]
second note:  I am following the process laid out in this thread:
https://github.com/orgs/OpenRadioss/discussions/1014

**Reply by MarianBulla** [reply]
Hi,
for runnung OpenRadioss using MPI on Windows, you need an external MPI executable.
you can get it here: https://www.intel.com/content/www/us/en/developer/tools/oneapi/mpi-library.html#gs.uq42jm
Then lounching the mpi.exe before engine...like this:

mpiexec.exe --map-by socket:PE=2 --bind-to core -n 2 engine_win64_impi.exe -i TWO_ELEMS_0000.rad

should work. 

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by danFrodo** [comment]
Hi Marian, it's Dan Fradl, but I bet you figured that out.  😊

I've been in meetings all morning, finally getting back on this.  That link
is what I installed but the install directory doesn't show up.  If I try to
reinstall the installer seems to think it is installed.  I don't know where
it put it.

I'm expecting it in "program files x86/intel" but directory isn't being
built.  It's really weird.

I was excited to see the nice script on the forum and will use that once I
get this weird intel install fixed.

**Comment by danFrodo** [comment]
I was just installing the runtime version.  Do I need to have visual
studio?  the whole API toolkit?  I only was installing this one thing
(which doesn't install properly even though the stupid wizard things it
does)

[image: image.png]

**Comment by danFrodo** [comment, marked as answer]
Looks like our workstation/compute server is getting delivered today so
maybe this laptop install won't matter anymore.  I am hoping to make the
new machine linux.

I asked management for either 50 or 72 altair tokens but it wasn't in
budget, so it was denied.  I came back saying bare minimum is 30 tokens and
they found some money.  So in a week or so at least I'll be official Altair
customer again, though at first as a very small account.
